Sources/PathKit.swift

```diff
@@ -608,6 +617,17 @@ extension Path : Sequence {
   public func makeIterator() -> DirectoryEnumerator {
     return DirectoryEnumerator(path: self)
   }
+
+  /// Perform a deep enumeration of a directory.
+  ///
+  /// - Parameter options: FileManager directory enumerator options.
+  ///
+  /// - Returns: a directory enumerator that can be used to perform a deep enumeration of the
+  ///   directory.
+  ///
+  public func makeIterator(options: FileManager.DirectoryEnumerationOptions) -> DirectoryEnumerator {
```

Comment:
I can do that. Any preference on how to call it? `Path.DirectoryEnumerationOptions`?